### PR TITLE
Pool info utils multicall

### PR DESCRIPTION
**Pull request by prat-gpt**
# Description of change
## High level
* Add `PoolInfoUtilsMulticall` contract to call multiple `PoolInfoUtils` methods in a single call to reduce rpc calls from subgraph

### CODE REVIEW

**Review by MikeHathaway — APPROVED** [review]
Thanks - this should be able to help streamline a bunch of calls

File: src/PoolInfoUtilsMulticall.sol [hunk 1]
```diff
@@ -0,0 +1,308 @@
+// SPDX-License-Identifier: BUSL-1.1
```
Comment (MikeHathaway) [inline src/PoolInfoUtilsMulticall.sol]: Should we change the license?

**Review by ith-harvey — APPROVED** [review]
LGTM

File: src/PoolInfoUtilsMulticall.sol (hunk 1 above)
Comment (ith-harvey) [reply src/PoolInfoUtilsMulticall.sol]: Updated -> https://github.com/ajna-finance/contracts/pull/944/commits/ff99d0374d93356bad27a2bc8f8d197b1ec8821e

**Pull request merged**